front/src/applications/stdcm/hooks/useStdcmTowedRollingStock.ts

```diff
@@ -0,0 +1,27 @@
+import { useSelector } from 'react-redux';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import type { StdcmConfSelectors } from 'reducers/osrdconf/stdcmConf/selectors';
+
+const useStdcmTowedRollingStock = () => {
+  const { getTowedRollingStockID } = useOsrdConfSelectors() as StdcmConfSelectors;
+  const towedRollingStockId = useSelector(getTowedRollingStockID);
+
+  const { data: towedRollingStock } =
+    osrdEditoastApi.endpoints.getTowedRollingStockByTowedRollingStockId.useQuery(
+      {
+        towedRollingStockId: towedRollingStockId!,
+      },
+      {
+        skip: !towedRollingStockId,
+      }
+    );
+
+  return {
+    towedRollingStockId,
+    towedRollingStock: towedRollingStockId ? towedRollingStock : undefined,
+  };
```

Comment:
We don't use `towedRollingStockId` anywhere

```suggestion
  return towedRollingStock: towedRollingStockId ? towedRollingStock : undefined;
```